packages/storybook/stories/va-button-uswds.stories.jsx

```diff
@@ -113,29 +116,34 @@ export const Disabled = Template.bind(null);
 Disabled.args = {
   ...defaultArgs,
   disabled: true,
-  text: "Default",
+  text: 'Default',
 };
 
 export const Loading = Template.bind(null);
 Loading.args = {
   ...defaultArgs,
   text: 'Click to load',
-  onClick: (e) => {
+  onClick: e => {
     e.target.setAttribute('text', '');
     e.target.setAttribute('loading', 'true');
     setTimeout(() => {
       e.target.setAttribute('text', 'Click to load');
       e.target.setAttribute('loading', 'false');
-    }, 5000)
-  }
-  
+    }, 5000);
+  },
 };
 
+export const FullWidth = Template.bind(null);
+FullWidth.args = {
+  ...defaultArgs,
+  fullWidth: true,
+  text: 'Default',
+};
```

Comment:
I don't see this working in the [Chromatic link](https://3644-full-width-buttons--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/docs/uswds-va-button--docs#full-width).

![Screenshot 2025-03-05 at 12 39 52 PM](https://github.com/user-attachments/assets/beef72d3-e3ca-4141-97de-9754b91ba60c)

Reply:
Should be working now.